functions-and-operators/string-functions.md

```diff
@@ -121,13 +121,92 @@ Return number of characters in argument.
 
 Synonym for `CHAR_LENGTH()`.
 
-### [`CONCAT()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_concat)
+### [`CONCAT(str1,str2,...)`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_concat)
 
-Return concatenated string.
+The `CONCAT()` function is used to concatenate one or more arguments.
 
-### [`CONCAT_WS()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_concat-ws)
+```sql
+SELECT CONCAT('TiDB', NULL, 'Server');
```

Comment:
I think it might be best to start with a simple common example instead of a exception like how it handles `NULL`.

So maybe first something like `SELECT CONCAT('foo','bar','baz')` before having an example with a `NULL` argument.